src/platform/datapath_raw_xdp_win.c

```diff
@@ -476,7 +478,8 @@ CxPlatDpRawInterfaceUninitialize(
     if (Interface->Rules != NULL) {
         for (uint8_t i = 0; i < Interface->RuleCount; ++i) {
             if (Interface->Rules[i].Pattern.IpPortSet.PortSet.PortSet) {
-                CxPlatFree(Interface->Rules[i].Pattern.IpPortSet.PortSet.PortSet, PORT_SET_TAG);
+                CxPlatFree(
+                    (uint8_t*)Interface->Rules[i].Pattern.IpPortSet.PortSet.PortSet, PORT_SET_TAG);
```

Comment:
Why was this cast necessary?

Reply:
It looks like MsQuic is using the rule to hold a pointer to the PortSet. This makes the MsQuic code clunky because XDP rules now take a `const` pointer to the port set.